### PR TITLE
docs: IFS-4064 rewrite reference architecture for backends

### PR DESCRIPTION
**Pull request by stdll**
### **User description**
- minimize redundancies with other content
- prune explanations which have become common knowledge
- update architecture diagrams
- move content about batches to their respective pages
- mark paragraphs that reference the backends docs as "content to be revised"
- replace term "Datenzugriffsschicht" with "Persistenzschicht"

___

### **PR Type**
Documentation

___

### **Description**
- Rewrote and restructured the backend documentation to minimize redundancies and update outdated content.
- Updated architecture diagrams across multiple documents to reflect the new backend structure.
- Introduced a new document detailing the backend reference architecture, including components and technical aspects.
- Revised terminology, replacing "Datenzugriffsschicht" with "Persistenzschicht" and updating glossary references.
- Removed outdated diagrams and content, focusing on current best practices and technologies.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><details><summary>13 files</summary><table>
<tr>
  <td>
    <details>
      <summary><strong>inhalt.adoc</strong><dd><code>Revise application core content and update diagrams</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

isyfact-standards-doc/src/docs/antora/modules/blaupausen/pages/detailkonzept-komponente-anwendungskern/inhalt.adoc

<li>Updated terminology and references to glossary terms.<br> <li> Replaced outdated diagrams with new ones.<br> <li> Revised descriptions of application core components.<br>

</details>

  </td>
  <td><a href="https://github.com/IsyFact/isyfact-standards/pull/555/files#diff-d5c9c51bc3df53310f864e49bc8d158ecb66634199a6d05ce1350f1dd0fb4414">+19/-17</a>&nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>thisdoc.adoc</strong><dd><code>Enhance introduction and remove redundancy</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

isyfact-standards-doc/src/docs/antora/modules/blaupausen/pages/detailkonzept-komponente-anwendungskern/thisdoc.adoc

<li>Enhanced introduction with additional details.<br> <li> Removed redundant overview section.<br>

</details>

  </td>
  <td><a href="https://github.com/IsyFact/isyfact-standards/pull/555/files#diff-67fc3ffaf65bb77313a06d6f2ed5bbee8b5caf6ea4a8833efcf59c01b0438959">+5/-25</a>&nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>thisdoc.adoc</strong><dd><code>Expand batch processing details and add diagram</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

isyfact-standards-doc/src/docs/antora/modules/blaupausen/pages/detailkonzept-komponente-batch/thisdoc.adoc

<li>Expanded explanation of batch processing.<br> <li> Added new diagram for batch system architecture.<br>

</details>

  </td>
  <td><a href="https://github.com/IsyFact/isyfact-standards/pull/555/files#diff-17732be0b08596e8f7f95e26be5d3ea449f5c1edd55530af6e12edf91b5aac24">+30/-3</a>&nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>einordnung-ziele.adoc</strong><dd><code>Update persistence layer description and add diagram</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

isyfact-standards-doc/src/docs/antora/modules/blaupausen/pages/detailkonzept-komponente-datenzugriff/einordnung-ziele.adoc

<li>Updated description of the persistence layer.<br> <li> Added new diagram for persistence system architecture.<br>

</details>

  </td>
  <td><a href="https://github.com/IsyFact/isyfact-standards/pull/555/files#diff-a51e855d50a05844c44ba6975cd369daddc79887276a080c2b99ba7cfb0fc1e6">+9/-9</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>inhalt.adoc</strong><dd><code>Update service architecture and terminology</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

isyfact-standards-doc/src/docs/antora/modules/blaupausen/pages/detailkonzept-komponente-service/inhalt.adoc

<li>Updated service architecture diagram.<br> <li> Revised descriptions to align with new backend terminology.<br>

</details>

  </td>
  <td><a href="https://github.com/IsyFact/isyfact-standards/pull/555/files#diff-b40b5baf0c073a3a5848ce2ca2d2ca1bd6a5ebf76d5f4a9ec81faef5ba96df67">+2/-2</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>thisdoc.adoc</strong><dd><code>Update backend service references</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

isyfact-standards-doc/src/docs/antora/modules/blaupausen/pages/detailkonzept-komponente-service/thisdoc.adoc

- Updated references to backend services.

</details>

  </td>
  <td><a href="https://github.com/IsyFact/isyfact-standards/pull/555/files#diff-34a71bb7dab70313cf622375b95850e0ba7c8ff11005aaeef75af4becbd661df">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>inhalt.adoc</strong><dd><code>Update IT system architecture diagram</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

isyfact-standards-doc/src/docs/antora/modules/einstieg/pages/tutorial/inhalt.adoc

- Updated IT system architecture diagram.

</details>

  </td>
  <td><a href="https://github.com/IsyFact/isyfact-standards/pull/555/files#diff-fdaed9c2901e55420649a84d2e1eb84b945f56b3b4a74f7edb083ced04e4d072">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>isyfact-standards.adoc</strong><dd><code>Introduce backend architecture and update references</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

isyfact-standards-doc/src/docs/antora/modules/einstieg/pages/vorstellung/isyfact-standards.adoc

<li>Introduced backend system architecture.<br> <li> Updated references to backend components.<br>

</details>

  </td>
  <td><a href="https://github.com/IsyFact/isyfact-standards/pull/555/files#diff-f073aaa8b1027c6fc6af5fb11faabd871a27d806c77b4501aea207e769cc1d7e">+12/-4</a>&nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>umsetzung-datenzugriff.adoc</strong><dd><code>Update backend persistence references</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

isyfact-standards-doc/src/docs/antora/modules/isy-persistence/pages/nutzungsvorgaben/umsetzung-datenzugriff.adoc

- Updated references to backend persistence architecture.

</details>

  </td>
  <td><a href="https://github.com/IsyFact/isyfact-standards/pull/555/files#diff-7d08c5facc48c70e00fb96176adbc484fbc0eb185e6a6d4e21df51ec61e660aa">+2/-2</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>inhalt.adoc</strong><dd><code>Update security references to backend</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

isyfact-standards-doc/src/docs/antora/modules/isy-security/pages/konzept/inhalt.adoc

- Updated security component references to backend.

</details>

  </td>
  <td><a href="https://github.com/IsyFact/isyfact-standards/pull/555/files#diff-351832cb9dafc22dce838fb7c20a7a0f7379b996f760e1975b0dc73bd705e5e5">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>backend.adoc</strong><dd><code>Add backend reference architecture document</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/software-technisch/backend.adoc

<li>Added new document for backend reference architecture.<br> <li> Detailed backend components and architecture.<br>

</details>

  </td>
  <td><a href="https://github.com/IsyFact/isyfact-standards/pull/555/files#diff-ec65b2631ef704caf12975d5c9ccb406e555163d264bba19ce9619c2d88890ea">+167/-0</a>&nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>changelog.adoc</strong><dd><code>Add changelog for backend architecture</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/software-technisch/backend/changelog.adoc

- Added changelog for backend reference architecture.

</details>

  </td>
  <td><a href="https://github.com/IsyFact/isyfact-standards/pull/555/files#diff-fd12af907e4e977ce3d9b1a2ba127667e0045626c1aa9a92fa0e8f555a8f2d35">+55/-1</a>&nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>konfiguration.adoc</strong><dd><code>Add backend configuration implementation details</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/software-technisch/backend/konfiguration.adoc

- Added section on backend configuration implementation.

</details>

  </td>
  <td><a href="https://github.com/IsyFact/isyfact-standards/pull/555/files#diff-1b2329e81550ca14140fd365bf2e09cfac6574e5041db03e218b5d083bc6f353">+1/-0</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    
</table></details></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

File: isyfact-standards-doc/src/docs/antora/modules/blaupausen/pages/detailkonzept-komponente-batch/thisdoc.adoc [hunk 1]
```diff
@@ -4,11 +4,38 @@
...
+Batches werden als eigener Prozess auf einem eigenen Server ausgeführt, das heißt sie laufen nicht in der virtuellen Maschine des Application Servers ab.
+Batches werden somit in einer eigenen Ablaufumgebung ausgeführt und greifen direkt auf die Datenbank oder auch auf Dateien zu.
+Die benötigte Funktionalität des Anwendungskerns wird dem Batch als Bibliothek zur Verfügung gestellt und nicht über einen Service-Aufruf genutzt.
+Der Grund für diese Entscheidung liegt in den Datenmengen, die normalerweise von einem Batch verarbeitet werden: Die Übermittlung dieser Datenmengen über eine remote genutzte Schnittstelle ist ein möglicher Flaschenhals in der Anwendung.
+Dieser Flaschenhals wird durch die Nutzung der Anwendungskernfunktionalität als Bibliothek vermieden.
+
+Batch-Abläufe bestehen aus einem oder mehreren Batch-Schritten.
+Die einzelnen Batch-Schritte werden von einem Scheduler aufgerufen und zum vollständigen Batch-Ablauf verbunden.
+Ein Batch-Schritt wird von einem Programm implementiert, das mit entsprechenden Parametern vom Scheduler aufgerufen werden kann.
+Die hier beschrieben Batches sind genau diese Batch-Schritte.
```
Comment (msg-jstren) [inline isyfact-standards-doc/src/docs/antora/modules/blaupausen/pages/detailkonzept-komponente-batch/thisdoc.adoc]: Grammatik
```suggestion
Die hier beschriebenen Batches sind genau diese Batch-Schritte.
```

File: isyfact-standards-doc/src/docs/antora/modules/blaupausen/pages/detailkonzept-komponente-service/inhalt.adoc [hunk 2]
```diff
@@ -167,7 +167,7 @@ Parameter dieser Meldung sind Vor- und Nachname sowie das Geburtsdatum.
 Dazu gibt es einen Meldung-Service in der Version 1.0. Dieser wird in der Service-Schicht des IT-Systems implementiert.
 Ab einem Stichtag soll zusätzlich noch das Geschlecht gemeldet werden.
 Im bisherigen Datenbestand wird dieses neue Attribut auf den Wert „unbekannt“ gesetzt.
-Der bestehende Service wird um dieses Attribut erweitert und erhält die Versionsnummer 1.1. Anwendungskern und Datenzugriffsschicht müssen ebenfalls erweitert werden.
+Der bestehende Service wird um dieses Attribut erweitert und erhält die Versionsnummer 1.1. Anwendungskern und Persistenzschicht müssen ebenfalls erweitert werden.
```
Comment (msg-jstren) [inline isyfact-standards-doc/src/docs/antora/modules/blaupausen/pages/detailkonzept-komponente-service/inhalt.adoc]: Satzende neue Zeile
```suggestion
Der bestehende Service wird um dieses Attribut erweitert und erhält die Versionsnummer 1.1.
Anwendungskern und Persistenzschicht müssen ebenfalls erweitert werden.
```

File: isyfact-standards-doc/src/docs/antora/modules/einstieg/pages/vorstellung/isyfact-standards.adoc [hunk 3]
```diff
@@ -52,16 +52,24 @@ Die Anforderungsliste ist ein Instrument, um die Übersicht und die Nachvollzieh
 
 Die xref:methodik:vorlagen.adoc#datenflussdiagramm[Vorlage zu Datenflussdiagrammen] beschreibt die Erstellung von Datenflussdiagrammen und liefert eine Leseanleitung dazu.
 
+
```
Comment (msg-jstren) [inline isyfact-standards-doc/src/docs/antora/modules/einstieg/pages/vorstellung/isyfact-standards.adoc]: Leere Zeile entfernen (Doppelleerzeile nicht notwendig)

File: isyfact-standards-doc/src/docs/antora/modules/isy-persistence/pages/nutzungsvorgaben/umsetzung-datenzugriff.adoc [hunk 4]
```diff
@@ -53,10 +53,10 @@ Der Zugriff auf die Datenbank aus dem xref:glossary:glossary:master.adoc#glossar
...
 Für jede Entität wird ein entsprechendes Repository als Interface angelegt.
 
-Die Repositories werden im Package der Fachkomponente abgelegt, welche die Datenhoheit über die Tabelle(n) des Repositories besitzt (zum Thema Datenhoheit siehe xref:blaupausen:referenzarchitektur-it-system/master.adoc#datenzugriff[Referenzarchitektur IT-System]).
+Die Repositories werden im Package der Fachkomponente abgelegt, welche die Datenhoheit über die Tabelle(n) des Repositories besitzt (zum Thema Datenhoheit siehe xref:referenzarchitektur:software-technisch/backend.adoc#persistenz[Referenzarchitektur Backend]).
 Die Repositories werden nur von Klassen der Fachkomponente mit Datenhoheit aufgerufen.
 
-Gemäß der xref:blaupausen:referenzarchitektur-it-system/master.adoc#datenzugriff[Referenzarchitektur] dürfen Entitäten nur von der Fachkomponente im Anwendungskern verändert werden, welche zur Fachkomponente im Datenzugriff gehört.
+Gemäß der xref:referenzarchitektur:software-technisch/backend.adoc#persistenz[Referenzarchitektur] dürfen Entitäten nur von der Fachkomponente im Anwendungskern verändert werden, welche zur Fachkomponente im Datenzugriff gehört.
```
Comment (msg-jstren) [inline isyfact-standards-doc/src/docs/antora/modules/isy-persistence/pages/nutzungsvorgaben/umsetzung-datenzugriff.adoc:59]: Zielt der Nebensatz auf die Entitäten ab (welche)? Dann Plural.
Satz ist aber schwer zu lesen..
```suggestion
Gemäß der xref:referenzarchitektur:software-technisch/backend.adoc#persistenz[Referenzarchitektur] dürfen Entitäten nur von der Fachkomponente im Anwendungskern verändert werden, welche zur Fachkomponente im Datenzugriff gehören.
```

File: isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/software-technisch/backend.adoc [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+
+In Backends ist der xref:blaupausen:detailkonzept-komponente-anwendungskern/master.adoc#transaktionssteuerung[Anwendungskern] für die Transaktionssteuerung zuständig.
+In Ausnahmefällen kann die Transaktionssteuerung auch über die xref:blaupausen:detailkonzept-komponente-service/master.adoc#transaktionssteuerung[Serviceschicht] erfolgen.
+
+=== Caching
+
+Anwendungsspezifische Caches sind mit der Spring Cache Abstraction und einem geeigneten Caching Provider zu implementieren.
+
+Die https://docs.spring.io/spring-framework/reference/integration/cache.html[Spring Cache Abstraction] bietet eine effiziente und flexible Möglichkeit, Caching in Anwendungen zu implementieren.
+Durch ihre Verwendung wird, z.B. über die Nutzung von Annotationen, das Hinzufügen von Caching zu bestehenden Methoden erleichtert.
```
Comment (msg-jstren) [inline isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/software-technisch/backend.adoc]: z.B. ausschreiben, wurde im Bedienkonzept komplett umgestellt, sollte hier folglich auch so sein.
```suggestion
Durch ihre Verwendung wird das Hinzufügen von Caching, zum Beispiel über die Nutzung von Annotationen, zu bestehenden Methoden erleichtert.
```

File: isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/software-technisch/backend.adoc [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+[[persistenz-fachkomponenten]]
+=== Fachkomponenten
+
+Die Fachkomponenten der Persistenzschicht entsprechen vom fachlichen Schnitt her denen des Anwendungskerns.
+Die Fachkomponenten des Anwendungskerns besitzen wiederum die Datenhoheit auf ihre "Entsprechungen" in der Persistenzschicht.
+Nur die Fachkomponente mit Datenhoheit darf Änderungen an den jeweiligen persistenten Entitäten vornehmen.
+
+.icon:university[title=Architekturregel] Persistente Entitäten im Anwendungskern
+****
+Persistente Entitäten dürfen nicht über den Anwendungskern hinaus herausgegeben werden. Sie haben im Anwendungskern zu verbleiben.
```
Comment (msg-jstren) [inline isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/software-technisch/backend.adoc]: Nur ein Satz pro Zeile (Antora Clean Code)
```suggestion
Persistente Entitäten dürfen nicht über den Anwendungskern hinaus herausgegeben werden.
Sie haben im Anwendungskern zu verbleiben.
```

File: isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/software-technisch/backend.adoc [hunk 7]
```diff
@@ -0,0 +1,167 @@
...
+
+Zur Konzeption und Umsetzung des Anwendungskerns definiert die Referenzarchitektur xref:blaupausen:detailkonzept-komponente-anwendungskern/master.adoc[weitere Vorgaben].
+
+[[persistenz]]
+== Persistenz
+
+include::blaupausen:detailkonzept-komponente-datenzugriff/einordnung-ziele.adoc[tags=persistenz-definition]
+
+[[persistenz-fachkomponenten]]
+=== Fachkomponenten
```
Comment (msg-jstren) [inline isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/software-technisch/backend.adoc:83]: Sollten die weiteren Persistenz-Abschnitte nicht auch hier abgebildet werden: blaupausen:detailkonzept-komponente-datenzugriff/einordnung-ziele.adoc[] und wie beim allgemeinen Begriff in dieses adoc includiert werden?
Betrifft die zwei Abschnitte [[persistenz-fachkomponenten]] und [[persistenz-framework]].

File: isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/software-technisch/backend.adoc [hunk 8]
```diff
@@ -0,0 +1,167 @@
...
+Zur Umsetzung des objekt-relationalen Mapping verwendet die Persistenzschicht das Produkt Hibernate.
+
+[[persistenz-umsetzung]]
+=== Umsetzung
+
+Zur Konzeption der Persistenzschicht definiert die Referenzarchitektur xref:blaupausen:detailkonzept-komponente-datenzugriff/einordnung-ziele.adoc[weitere Vorgaben].
+Die Umsetzung geschieht mithilfe des xref:isy-persistence:konzept.adoc[Bausteins JPA/Hibernate].
+
+[[service]]
+== Service
```
Comment (msg-jstren) [inline isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/software-technisch/backend.adoc:112]: Sollten diese Abschnitte nicht eher hier beschrieben werden: isyfact-standards-doc/src/docs/antora/modules/blaupausen/pages/detailkonzept-komponente-service/inhalt.adoc und in diesem adoc nur includiert werden?
Betrifft die drei Abschnitte [[service]], [[service-fachkomponenten]] und [[service-framework]].

File: isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/software-technisch/backend.adoc [hunk 9]
```diff
@@ -0,0 +1,167 @@
...
+[[service-fachkomponenten]]
+=== Fachkomponenten
+
+Die Servicelogik beinhaltet, wie im Anwendungskern auch, Fachkomponenten.
+Diese Fachkomponenten setzen im wesentlichen Service-Schnittstellen mit den folgenden Aufgaben um:
+
+* Autorisierung und Tracing eingehender Anfragen,
+* Validierung der Transportobjekte,
+* Umwandlung von Transportobjekten in Geschäftsobjekte und umgekehrt,
+* fachliche Fehlerbehandlung.
```
Comment (msg-jstren) [inline isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/software-technisch/backend.adoc:130]: Würde die Satzzeichen am Ende der Stichpunkte alle entfernen
```suggestion
* Autorisierung und Tracing eingehender Anfragen
* Validierung der Transportobjekte
* Umwandlung von Transportobjekten in Geschäftsobjekte und umgekehrt
* fachliche Fehlerbehandlung
```

File: isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/software-technisch/backend.adoc [hunk 10]
```diff
@@ -0,0 +1,167 @@
...
+Vorgaben und Umsetzung mithilfe des xref:isy-exception-core:konzept/master.adoc[Bausteins Fehlerbehandlung]
+
+Konfiguration::
+xref:software-technisch/backend/konfiguration.adoc#konfiguration-arten[Vorgaben] aus der Referenzarchitektur, xref:software-technisch/backend/konfiguration.adoc#umsetzung[Umsetzung] mithilfe von Spring
+
+Logging & Tracing::
+Vorgaben und Umsetzung mithilfe des xref:isy-logging:konzept/master.adoc[Bausteins Logging]
+
+Autorisierung::
+Vorgaben und Umsetzung mithilfe des xref:isy-security-doc::konzept/master.adoc[Bausteins Security]
```
Comment (msg-jstren) [inline isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/software-technisch/backend.adoc:165]: Lässt sich nicht öffnen; Verweist das schon auf die ausgelagerte Version von isy-security?
```suggestion
Vorgaben und Umsetzung mithilfe des xref:isy-security-doc::konzept/master.adoc[Bausteins Security]
```

File: isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/software-technisch/backend/changelog.adoc
Comment (msg-jstren) [inline isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/software-technisch/backend/changelog.adoc:1]: Wird diese Changelog-Datei benötigt?

Comment (stdll) [reply isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/software-technisch/backend/changelog.adoc:1]: Die Datei wird mit einem bereits geplanten Epic konsolidiert und musste daher übernommen werden.

File: isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/software-technisch/backend.adoc (hunk 10 above)
Comment (stdll) [reply isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/software-technisch/backend.adoc:165]: Genau! Ich habe den Verweis getestet.

File: isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/software-technisch/backend.adoc (hunk 9 above)
Comment (stdll) [reply isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/software-technisch/backend.adoc:130]: Laut den [Interpunktionsregeln bei Aufzählungszeichen](https://de.wikipedia.org/wiki/Aufz%C3%A4hlungszeichen#Interpunktionsregeln) (und ich habe es so gelernt): "Grundsätzlich werden aus Sicht der Interpunktionsregeln Aufzählungszeichen so behandelt, als seien sie nicht vorhanden. Das heißt, dass Interpunktion so gesetzt werden muss, als gäbe es keine typografische Gliederung."

File: isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/software-technisch/backend.adoc (hunk 8 above)
Comment (stdll) [reply isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/software-technisch/backend.adoc:112]: Genau das erfolgt in den Nachfolgetickets, die sich mit den jeweiligen Unterseiten befassen. Das, was du beschreibst, ist dort Teil der Abnahmenkriterien.

File: isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/software-technisch/backend.adoc (hunk 7 above)
Comment (stdll) [reply isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/software-technisch/backend.adoc:83]: Genau das erfolgt in den Nachfolgetickets, die sich mit den jeweiligen Unterseiten befassen. Das, was du beschreibst, ist dort Teil der Abnahmenkriterien.

File: isyfact-standards-doc/src/docs/antora/modules/isy-persistence/pages/nutzungsvorgaben/umsetzung-datenzugriff.adoc (hunk 4 above)
Comment (stdll) [reply isyfact-standards-doc/src/docs/antora/modules/isy-persistence/pages/nutzungsvorgaben/umsetzung-datenzugriff.adoc:59]: Der ist so korrekt. Der Nebensatz bezieht sich auf die "Fachkomponente im Anwendungskern".

**Review by msg-jstren — APPROVED** [review]

**Pull request merged**